UnitWorld_server/server/networking/ClientsGameSender.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include "PlayerClient.h"
+
+#include "shared/game/GameManager.h"
+
+#include "shared/transfers/PhysicsCommunicationAssembler.h"
+
+#include "shared/communication/MessageSerializer.h"
+#include "shared/communication/messages/CompleteGameStateMessage.h"
+
+#include "shared/communication/messages/game/CommunicatedPlayer.h"
+#include "shared/communication/messages/game/CommunicatedSinguity.h"
+
+#include "communications/CommunicationHandler.h"
+
+#include <chrono>
+#include <mutex>
+
+namespace uw
+{
+    class ClientsGameSender
+    {
+    public:
+        ClientsGameSender(std::shared_ptr<GameManager> gameManager, std::shared_ptr<MessageSerializer> messageSerializer, std::shared_ptr<PhysicsCommunicationAssembler> physicsCommunicationAssembler) :
+            _gameManager(gameManager),
+            _messageSerializer(messageSerializer),
+            _maxMsBetweenCompleteStates(60000 / MINIMAL_COMPLETE_STATE_PER_MINUTE),
```

Comment:
Same thing here as for the `1000 / ...` constant

Reply:
do you Also insist for a timing helper class here?

Reply:
nah, not worth the maintainability cost yet